composer.json

```diff
@@ -38,7 +38,10 @@
         "integrated/puphpet": "~0.4",
         "integrated/content-history-bundle": "~0.6,<0.8",
         "integrated/comment-bundle": "~0.6,<0.8",
-        "integrated/thompson-theme-bundle": "~0.7.0"
+        "integrated/thompson-theme-bundle": "~0.7.0",
+        "antimattr/mongodb-migrations-bundle": "^1.0",
+        "doctrine/doctrine-migrations-bundle": "^1.2",
+        "doctrine/migrations": "1.2.2"
```

Comment:
You only need to require, i think:

"antimattr/mongodb-migrations-bundle": "^1.0",
"doctrine/doctrine-migrations-bundle": "^1.2",